library/components/TPdfDownloadHelper.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div v-if="isVisible" id="seleniumComponent" class="fixed top-0 bottom-0" />
+</template>
+
+<script>
+export default {
+  data: () => ({
+    responses: 0,
+    isVisible: false,
+  }),
+  watch: {
+    responses: _.debounce(function (newVal) {
+      this.showSeleniumComponent();
+    }, 1000),
```

Comment:
This would just wait for no ajax requests for 1 second?

Reply:
Yes, basically ajax keeps sending requests to pdf worker for response at the interval of 500ms until it receives 200, so this component is using debounce to wait until there is no request within 1 second and it will stop and show the component for selenium.

Reply:
this could include other non-relevant requests too, like analytics or something

Reply:
Just checked it does not include them because we are just intercepting axios (which is used internally) but if there is another call added to axios that is not part of layers, then it would cause problems, so I will just filter out layers request only, thanks for pointing out.